### PR TITLE
Make report calculating hour markings related to learnings

### PR DESCRIPTION
**Pull request by neurogoo**
There is Test category for all the rest hours for debugging. Still will be moved before going to prod.

### CODE REVIEW

File: reports-app/src/Futurice/App/Reports/DoWeStudy.hs [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            th_ "Project"
+            th_ "Hours"
+            th_ "Description"
+
+        tbody_ $ for_ (sortBy (comparing dwsName <> comparing dwsDate) xs) $ \DoWeStudy {..} -> tr_ $ do
+            td_ [ class_ "nowrap" ] $ toHtml $ textShow dwsDate
+            td_  [ class_ "nowrap" ] $ toHtml dwsName
+            td_ $ toHtml dwsTribe
+            td_ $ case dwsCategory of
+              Jedi -> toHtml ("Jedi" :: Text)
```
Comment (phadej) [inline reports-app/src/Futurice/App/Reports/DoWeStudy.hs]: define `ToHtml TrainingType` instance

File: reports-app/src/Futurice/App/Reports/DoWeStudy.hs [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+              OtherTraining -> toHtml ("General training" :: Text)
+              Test -> toHtml ("Test" :: Text)
+            td_ $ toHtml dwsProject
+            td_ $ toHtml dwsHours
+            td_ $ wordsToHtml (T.words dwsDesc)
+  where
+    wordsToHtml [] = pure ()
```
Comment (phadej) [inline reports-app/src/Futurice/App/Reports/DoWeStudy.hs]: at least leave a note that this is copy of function from the other report

File: reports-app/src/Futurice/App/Reports/DoWeStudy.hs [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+{-# LANGUAGE MultiWayIf        #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE TemplateHaskell   #-}
+{-# LANGUAGE TypeFamilies      #-}
+module Futurice.App.Reports.DoWeStudy (
+    DoWeStudy (..),
+    DoWeStudyData (..),
+    doWeStudyData
+                                      ) where
```
Comment (phadej) [inline reports-app/src/Futurice/App/Reports/DoWeStudy.hs]: weird indentation

File: reports-app/src/Futurice/App/Reports/DoWeStudy.hs [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        trs <- PMQ.timereports interval uid
+
+        for (toList trs) $ \tr -> do
+            task <- PMQ.task (PM.trTask tr)
+            prj <- traverse PMQ.project (PM.taskProject task)
+            let comment = fromMaybe "<empty>" $ PM.trComment tr
+            let commentWs = Set.fromList $ T.words $ T.toLower comment
+            let otherTrainingSet = setOf (L.values . L._String) doWeStudyConfig
+
+            if | Set.member "jedi" commentWs -> do
```
Comment (phadej) [inline reports-app/src/Futurice/App/Reports/DoWeStudy.hs]: all cases return `Just`, is it on purpose?

Comment (neurogoo) [reply reports-app/src/Futurice/App/Reports/DoWeStudy.hs]: This is for debugging purposes. As I wrote, I will filter away non-learning ones before going to prod when I have locked what keywords I will be using

**Pull request merged**